utils/getFeed.js

```diff
@@ -107,6 +109,10 @@ function getFeedItems( parsed, nsResolver, maxItems, parseFeedItem ) {
 export function parseFeed( raw, maxItems = 0, parseFeedItem ) {
 	return new Promise( function( resolve, reject ) {
 		try {
+			// Use Wicked Good XPath for browsers that don't support XPath evaluation.
+			if ( "evaluate" in document === false ) {
```

Comment:
For this PR I think this is the way to go. Especially since we're also including `fetch` as a dependency.

But we should definitely consider expecting these things to simply exist. `wp_get_script_polyfill` could easily be expanded to conditionally load `wicked-good-xpath` only when necessary.